CachyOS Kernel Manager Turkish localisation

Adjudicator069 opened the pull request:
Any feedback or criticism is valued.

Adjudicator069 committed:
CachyOS Kernel Manager Turkish localisation

Any feedback or criticism is valued.

vnepogodin reviewed (approved):
LGTM

xeome commented on lang/cachyos-kernel-manager_tr.ts:
```diff
@@ -0,0 +1,272 @@
+    </message>
+    <message>
+        <source>Tweak kernel options prior to a build via nconfig</source>
+        <translation>nconfig aracılığıyla yapılandırmadan önce çekirdek ayarlarını iyileştir</translation>
```
"ayarlarına ince ayar yap" instead of "ayarlarını iyileştir"  is more suitable for this context in my opinion. Same goes with other translation where "iyileştir" was used.

Adjudicator069 replied:
'ince ayar yapma' is more equivalent of 'to tune' or 'finely tune' than 'to optimise', so I disagree.

xeome commented on lang/cachyos-kernel-manager_tr.ts:
```diff
@@ -0,0 +1,272 @@
+    </message>
+    <message>
+        <source>Build the ZFS module</source>
+        <translation>ZFS modülünü yapılandır</translation>
```
"ZFS modülünü derle" is more suitable.

Adjudicator069 replied:
'derle' means 'to compile', and to (re)build, you have to compile as part of the process that is building, which is why I used 'yapılandırma' instead of 'derleme'. Though, I suppose they colloquially refer to the same thing, and are somewhat interchangeable. Nevertheless, here's an educational source I used as a reference, not including a few Turkish forum and blog posts that I encountered during my scouring of the internet:

https://bim.hku.edu.tr/bilgisayar-terimler-sozlugu/

xeome commented on lang/cachyos-kernel-manager_tr.ts:
```diff
@@ -0,0 +1,272 @@
+    </message>
+    <message>
+        <source>Build the BCACHEFS module</source>
+        <translation>BCACHEFS modülünü yapılandır</translation>
```
same here, "derle" is more suitable instead of "yapılandır"

xeome commented on lang/cachyos-kernel-manager_tr.ts:
```diff
@@ -0,0 +1,272 @@
+    </message>
+    <message>
+        <source>Build the NVIDIA module</source>
+        <translation>NVIDIA modülünü yapılandır</translation>
```
"derle" instead of "yapılandır"

xeome commented on lang/cachyos-kernel-manager_tr.ts:
```diff
@@ -0,0 +1,272 @@
+    <name>ConfWindow</name>
+    <message>
+        <source>CachyOS Kernel Manager Configure</source>
+        <translation>CachyOS Çekirdek Yöneticisi Biçimleme</translation>
```
"Yapılandır" instead of "Biçimleme"

Adjudicator069 replied:
I also pondered on whether I should use "ayarla" or "yapılandır". An amenable proposition.

xeome commented:
Rest LGTM, translation wise.

Adjudicator069 committed:
Finalised cachyos-kernel-manager_tr.ts

Minor tweaks like rewording sentences and adjusting a few agent and locatum verbs for proper and common nouns to which they're subject to, wherever they're allowed, in order for them to sound more natural in the target language, without unnecessary pointers and symbols like asterisks and hyphens.

Adjudicator069 committed:
Minor update: cachyos-kernel-manager_tr.ts

One of the translated sections in my draft was missing from the pushed iteration for some reason. And the letter '-n' was missing from "<translation>RC…" as well.
